README.md

```diff
@@ -161,6 +161,39 @@ FreshRSS supports further customizations by adding extensions on top of its core
 See the [repository dedicated to those extensions](https://github.com/FreshRSS/Extensions). 
 
 
+# APIs & native apps
+
+FreshRSS supports access via native apps for Linux, Android, iOS and OS X.
+
+## Google Reader-like API
+
+There is more information available about our Google Reader compatible API on the page [mobile access](https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html).
+
+Supported clients are:
+
+* Android
+	* [News+](https://play.google.com/store/apps/details?id=com.noinnion.android.newsplus) with [News+ Google Reader extension](https://play.google.com/store/apps/details?id=com.noinnion.android.newsplus.extension.google_reader) (Closed source)
+	* [FeedMe 3.5.3+](https://play.google.com/store/apps/details?id=com.seazon.feedme) (Closed source)
+	* [EasyRSS](https://github.com/Alkarex/EasyRSS) (Open source, [F-Droid](https://f-droid.org/packages/org.freshrss.easyrss/))
+* GNU/Linux
+	* [FeedReader 2.0+](https://jangernert.github.io/FeedReader/) (Open source)
+
+## Fever API
+
+You can find more information about this API at our [Fever API documentation](https://freshrss.github.io/FreshRSS/en/users/06_Fever_API.html) page.
+
+Supported clients are:
+
+- iOS
+  - Fiery Feeds
```

Comment:
Could we have links? I believe the first one is https://itunes.apple.com/app/fiery-feeds-rss-reader/id1158763303 for instance

Reply:
I can see that Fiery Feeds already supports the Google Reader API. We could try to ask the developer to just add the possibility of typing a server address (like I did for FeedMe recently, which was in the same situation).

Reply:
How can you see that? The APP has nothing in "add account" that sounds like Google Reader. Does one of the supported services use GReade API in the background?

Reply:
If you are fine with linking to paid apps, I will add the links

Reply:
Feedly, The Old Reader, etc. are using the Google Reader API. This is how the other clients supporting FreshRSS are working.

Reply:
I do not have anything Apple around, so could you please check for which services Fiery Feeds gives the user the option to type a server address?

Reply:
Thanks for the info. I am writing to the developers of those 3 apps, to hear whether they could enable a generic service using Google Reader API.